### PR TITLE
inputplug: new, 0.4.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

- inputplug: new, 0.4.0

Package(s) Affected
-------------------

- inputplug: 0.4.0

Security Update?
----------------

No

Build Order
-----------

```
#buildit inputplug
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

File: app-utils/inputplug/autobuild/beyond [hunk 1]
```diff
@@ -0,0 +1,3 @@
+abinfo "Installing manpages ..."
+install -Dvm644 "$SRCDIR"/inputplug.1 \
+	/usr/share/man/man1/
```
Comment (MingcongBai) [inline app-utils/inputplug/autobuild/beyond]: Install to PKGDIR!

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by eatradish** [comment]
@aosc-buildit-bot build

**Comment by eatradish** [comment]
@aosc-buildit-bot build

**Comment by eatradish** [comment]
@aosc-buildit-bot build

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**